package mpsqual;

message RTCRequest
{
	enum RTCRequestT
    {
		RTC_GET	   = 0;
		RTC_SET	   = 1;
		RTC_SYSTEM_SET = 2;
		RTC_TO_SYSTEM  = 3;
		SYSTEM_TO_RTC  = 4;
	}

	required RTCRequestT requestType = 1;
	optional string		 timeString  = 2;
}

message RTCResponse
{
	required bool	success 	= 1;
	required string	timeString 	= 2;
}